### ISSUE TITLE
Slapping Bedrock with flint does nothing

### ISSUE THREAD

**Issue by Leno2206** [FTBTeam/FTB-Interactions#693]
## Modpack Version: 

v1.7.0
## Bug Report:

Slapping Bedrock with flint does nothing
## Expected Behavior/Suggestions:

Slapping bedrock with flint should result in Infinity dust
//

## Is It Repeatable? Steps to Reproduce: 

1. Take flint
2. Slap Bedrock/I do have more than 4xp
3. Get nothing

**Comment by Neeve01** [comment]
Are you sure you have enough XP? There’s also a small chance of not getting dust.

**Comment by Saereth** [comment]
need steps to reproduce this, its working fine for me in dev and 1.7 release

**Comment by Giovanni-NL** [comment]
Works for me in 1.7.0  
Perhaps @Leno2206 is right-clicking instead of left-clicking?

**Comment by Saereth** [comment]
marking this invalid unless you can show us a video of it not working for you and/or give steps to reproduce

**Comment by DeltaGeek** [comment]
I ran into this issue as well. It appears that if you don't have enough xp in your current level the dust won't scavenge even if you have multiple levels of xp.

In my case, I was partially into level 11 of xp and was unable to scavenge infinity dust. I ate two lotus blossoms and the scavenging started working again as I dropped 6 levels into getting 20 dust.

Hope this helps the original reporter and sheds some light on this edge case.
